app/logic/participants.py

```diff
@@ -96,23 +96,31 @@ def unattendedRequiredEvents(program, user):
 def getEventParticipants(event):
     eventParticipants = (EventParticipant
         .select()
-        .where(EventParticipant.event==event))
+        .where(EventParticipant.event == event))
 
     return {p.user.username: p.hoursEarned for p in eventParticipants}
 
-def getUserParticipatedEvents(program, user):
+def getUserParticipatedEvents(program, user, currentTerm):
     """
     This function returns a dictionary of all trainings for a program and
     whether the current user participated in them.
 
     :returns: trainings for program and if the user participated
     """
-    programTrainings = Event.select().join(ProgramEvent).where(Event.isTraining == 1, ProgramEvent.program == program)
+    ayStart = currentTerm.academicYearStartingTerm
+
+    programTrainings = (Event.select()
+                               .join(ProgramEvent)
+                               .where(Event.isTraining == True,
+                                      ProgramEvent.program == program,
+                                      (Event.term == ayStart) | (Event.term == currentTerm))
```

Comment:
If our current term is Summer, this will query will only be checking Fall and Summer

Reply:
Resolved